app/core/domain/email_manager.py

```diff
@@ -13,7 +13,7 @@
 # limitations under the License.
 
 """Config functions for managing email notifications."""
-
+# Check.
```

Comment:
Is this intended?

Reply:
That was just for testing I will delete it.

Reply:
Thanks -- once you fix this we can merge it!

Reply:
Done @seanlip